common.vars.pkr.hcl

```diff
@@ -1 +1 @@
-postgres-version = "15.1.0.137"
+postgres-version = "15.1.0.138-wrappers"
```

Comment:
We're no longer merging suffixes to develop - you can build an image on this branch and test if needed

Reply:
sure